### PR TITLE
Add Tmux talk

### PR DESCRIPTION
**Pull request by Advill**
Finally getting this thing merged. Might need some grammar changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ This is a collection of presentation notes from previous UUG meetings.
 * [Manpages](Manpages.md)
 * [Vim](https://crosse.github.io/vim_tutorial/) (External)
 * [Vim + Markdown](Vim.md)
+* [Tmux](tmux.md)
```
Comment (ripleymj) [inline README.md]: Alphabetize plz

File: tmux.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
+# TMUX
+## What is tmux?
+Tmux stands for terminal multiplexer. Anyone who has taken 261 might recognize
+the word multiplexer, but just in case you dont; multiplexers take one thing and
+split it into many. In 361 it's refering to circuts, here we're refering to your
```
Comment (ripleymj) [inline tmux.md]: "circuits" and "referring" x2

File: tmux.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+Both of these problems are easily solved with tmux, and they dont even require
+much technical overhead.
+
+## Using Tmux
+To start tmux, simply type `tmux` on the command line.
+
+Wow! A bottom bar!
+
+What you have now is simply a regular terminal running inside what Tmux calls
+a 'session'. sessions contian windows and windows contain terminals. None of
```
Comment (ripleymj) [inline tmux.md]: "contain"

File: tmux.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+
+Both of these problems are easily solved with tmux, and they dont even require
+much technical overhead.
+
+## Using Tmux
+To start tmux, simply type `tmux` on the command line.
+
+Wow! A bottom bar!
+
+What you have now is simply a regular terminal running inside what Tmux calls
```
Comment (ripleymj) [inline tmux.md]: You've got mixed "tmux" and "Tmux" throughout

File: tmux.md [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+a 'prefix' hotkey system, where `CTRL` + `b` is the default 'prefix' input. This
+simply means that (by default) tmux wont pay attention to any keystroke not
+preceded by the prefix.
+
+So now that you are sitting there with two terminals, how do you get between
+them?
+
+Start with your prefix, and follow it with the left arrow key.
+
+And like magic, your cursor moves to the left terminal. this works for all arrow
```
Comment (ripleymj) [inline tmux.md]: Capital "this"

File: tmux.md [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+keys exactly like you would imagine.
+
+To split your current terminal veritcally, simply enter the prefix followed by
```
Comment (ripleymj) [inline tmux.md]: "vertically"

File: tmux.md [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+`"`. This makes it easy to remember which one splits which way because they
+divide the keyboard in the direction they divide your terminal.
+
+Try splitting the terminal a couple times and moving around with the arrow keys.
+
+Now I'm sure you've opened an ungodly number of terminals and are wondering how
+to close them, well prefix + `x` has you there, will close the current terminal
+after a confirmation window pops up on the bottom in a different color. type `y`
+if you are sure you want your window closed. you can also type `exit` directly
```
Comment (ripleymj) [inline tmux.md]: Capital "you"

File: tmux.md [hunk 8]
```diff
@@ -0,0 +1,173 @@
...
+
+Quick ref:
+* prefix + `%` : split horizontally
+* prefix + `"` : split vertically
+* prefix + `x` : close terminal
+
+## Opening new windows
+To open a new window in tmux, type prefix followed by `c`. You will be greeted
+with a new terminal as well as a new entry in the bottom information bar.
+this bar shows open windows and their names. By default the names are generated
```
Comment (ripleymj) [inline tmux.md]: Capital "This"

File: tmux.md [hunk 9]
```diff
@@ -0,0 +1,173 @@
...
+to return to them later 
+
+To list all current sessions use prefix + `s`. If you have attempted to use tmux
+before you might even see a few sessions open! You can hit enter on the current
+attatched one to re-enter where you left off.
+
+If you do you can type prefix + `:`, which will open tmux command entry, and 
+type `kill-session -a`. This will simply kill all sessions besides the current.
+
+Lets detatch from the current session. Enter the prefix followed by `d` to
```
Comment (ripleymj) [inline tmux.md]: "detach"

File: tmux.md [hunk 10]
```diff
@@ -0,0 +1,173 @@
...
+manually detatch from the current session, you will return to the command line
```
Comment (ripleymj) [inline tmux.md]: "detach"

File: tmux.md [hunk 11]
```diff
@@ -0,0 +1,173 @@
...
+with a line of output from tmux that we don't particularly care about.
+
+Now open tmux again. You will be greeted with a new tmux session. To move to 
+a different session while already in a tmux session you can do prefix + `s` and
+move to the old one and hit enter to attach.
+
+These sessions will stay open once detatched until they are manually closed.
```
Comment (ripleymj) [inline tmux.md]: "detached"

File: tmux.md [hunk 12]
```diff
@@ -0,0 +1,173 @@
...
+* `:kill-session -a` : kill all other sessions besides the currently open one.
+
+### When are they useful?
+One last quick sidenote, sessions are automatically kept if a terminal is lost.
+This is most useful when stu or your internet are being dumb, and you get
+disconnected. Your session will be automatically kept and can be accessed by
+ssh-ing again or on the lab computers.
+
+### How can they hurt me?
+lets say you keep losing connection to stu, but never re-attach your session.
```
Comment (ripleymj) [inline tmux.md]: Capital "lets"

Comment (Crosse) [reply tmux.md]: Also consider "Let's", with an apostrophe. 😄

**Pull request merged**